lib/certs.go

```diff
@@ -201,7 +201,7 @@ func readCertsFromStream(reader io.Reader, filename string, format string, passw
 		for scanner.Scan() {
 			block, _ := pem.Decode(scanner.Bytes())
 			block.Headers = mergeHeaders(block.Headers, headers)
-			err := callback(block)
+			err := callback(block, format)
```

Comment:
nit: I think it's cleaner to pass the trimmed format here. I.e. "PEM" instead of whatever format contains.